File: iterableapi/src/main/java/com/iterable/iterableapi/IterableDefaultInAppHandler.java
package com.iterable.iterableapi;

import androidx.annotation.NonNull;

import com.iterable.iterableapi.messaging.inapp.interfaces.IterableInAppHandler;

public class IterableDefaultInAppHandler implements IterableInAppHandler {
    @NonNull
    @Override
    public InAppResponse onNewInApp(@NonNull IterableInAppMessage message) {
        return InAppResponse.SHOW;
    }
}

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableDefaultInAppHandler.kt
package com.iterable.iterableapi

import androidx.annotation.NonNull

class IterableDefaultInAppHandler : IterableInAppHandler {
    @NonNull
    override fun onNewInApp(@NonNull message: IterableInAppMessage): IterableInAppHandler.InAppResponse {
        return IterableInAppHandler.InAppResponse.SHOW
    }
}
